common/djangoapps/student/models.py

```diff
@@ -1320,7 +1312,7 @@ def username(self):
 
     @property
     def course(self):
-        return modulestore().get_course(self.course_id)
+        return CourseOverview.get_from_id(self.course_id)
 
```

Comment:
@mekkz thanks for the suggestion. I'd rather not pull on an additional, potentially very long thread as part of this PR, so I've marked the `course` property as deprecated and made a wrapper around `course_overview`.